test/sim/team-validator/breeding.js

```diff
@@ -233,6 +233,7 @@ describe('Team Validator', function () {
 		team = [
 			{species: 'ursalunabloodmoon', ability: 'mindseye', moves: ['yawn', 'bellydrum'], evs: {hp: 1}},
 			{species: 'greninjabond', ability: 'battlebond', moves: ['counter', 'switcheroo'], evs: {hp: 1}},
+			{species: 'pikachualola', ability: 'static', moves: ['wish', 'fakeout'], evs: {hp: 1}},
```

Comment:
Does this not fix the other two cases as well? (If it does then shouldn't you remove the learnset changes from #9888?)

Reply:
It does, but I wanted to make sure it's fine to go ahead and remove the learnset changes. It seems like there's no objection here, so I will go ahead and do that.